.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,104 @@
+name: publish distributions
+on:
+  push:
+    # branches:
+    # - main
+    # tags:
+    # - v*
```

Comment:
This was just commented out for me to verify things worked on my fork. I'll finish this soon and clean up and add context in PR body.